common/src/main/java/com/wynntils/utils/mc/McUtils.java

```diff
@@ -104,4 +105,12 @@ public static void sendSequencedPacket(PredictiveAction predictiveAction) {
     public static void sendChat(String message) {
         mc().getConnection().sendChat(message);
     }
+
+    /**
+     * Closes invisible containers opened in the background, without closing the visible screen.
+     */
+    public static void closeBackgroundContainer() {
+        player().connection.send(new ServerboundContainerClosePacket(McUtils.player().containerMenu.containerId));
+        player().containerMenu = McUtils.player().inventoryMenu;
+    }
```

Comment:
Maybe fits into container utils more, this is not so commonly used to be in McUtils